modules/react/button/stories/button/examples/Hyperlink.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+import {Hyperlink} from '@workday/canvas-kit-react/button';
+
+export const Link = () => <Hyperlink href="#">Hyperlink</Hyperlink>;
```

Comment:
Is it better to do it this way and have 'link' in the storybook nav or 
```
import {Hyperlink as Link} from '@workday/canvas-kit-react/button';

export const Hyperlink = () => <Link href="#">Hyperlink</Link>;
```
To have nav match the component name but have a more convoluted code to copy